### PR TITLE
Stickers/emojis fall out / improvements

### LINKED ISSUE #149: Emote error logs
2 errors from toggling the emote selector.

image

image

### LINKED ISSUE #179: Stickers/emojis fall out / improvements
- [x] Improve LBC sticker flow / clarity 
- [x] Show inline error if custom sticker amount is below min
- [x] improve quality of emojis

### PR DESCRIPTION
**Pull request by saltrafael**
testing: https://salt.odysee.com

## Fixes

Issue Number: Closes #179, Closes #149

## PR Checklist

<details><summary>Toggle...</summary>

What kind of change does this PR introduce?

- [X] Bugfix
- [X] Feature
- [ ] Code style update (formatting)
- [X] Refactoring (no functional changes)
- [ ] Documentation changes
- [ ] Other - Please describe:

### CODE REVIEW

**Comment by tzarebczan** [comment]
- [ ] fix sizing on livestreams

**Comment by infinite-persistence** [comment]
- [ ] https://salt.odysee.com/@dev-test:8/ls-1:e superchat avatars blinking when scrolling normal chat area.

**Review by infinite-persistence — COMMENTED** [review]
:+1:

**Comment by infinite-persistence** [comment]
- [ ] This button doesn't work
    - image

**Comment by tzarebczan** [comment]
thanks for the review @infinite-persistence ! @saltrafael please take a look and let's try to get this out today. 

Also if they are easy: 

- [ ] Change "fiat" to "Cash". 
- [ ] default to LBC tip if creator can't receive USD

File: ui/util/remark-emote.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { EMOTES_24px as EMOTES } from 'constants/emotes';
+import { EMOTES_72px as EMOTES } from 'constants/emotes';
 import visit from 'unist-util-visit';
 
 const EMOTE_NODE_TYPE = 'emote';
```
Comment (infinite-persistence) [inline ui/util/remark-emote.js:4]: Not a blocker, but there seems to be an error in the exchange rate calculation.
Also, the sentence feels inverted when in LBC mode.  Perhaps "Amount of <whatever entered> LBC in USD is lower than the price of $1" would flow better?

image

**Review by infinite-persistence — COMMENTED** [review]
Nice new stickers icon :+1:

File: ui/util/remark-emote.js (hunk 1 above)
Comment (saltrafael) [reply ui/util/remark-emote.js:4]: 👍🏻  great correction

**Pull request merged**